packages/code-du-travail-frontend/src/outils/common/CCSearchInfo.tsx

```diff
@@ -14,7 +14,9 @@ const CCSearchInfo: React.FC<Props> = ({ ccn }) => {
       {ccn && (
```

Comment:
Ici il ne faut afficher le message que si ccn.selected is defined non ?

Reply:
Pour les ancien simulateur, la donnée est toujours dans `ccn`. Je n'ai pas trouvé utile de la bouger car ça complexifiait pas mal de chose et on va passer sur la nouvelle step bientôt.